### PR TITLE
Try to save a stable build with symlink

### PR DESCRIPTION
**Pull request by CoryMartin-NOAA**
Updating the stable-nightly branch update cron script with the capability (hopefully) to a fixed symlink to the most recent stable build on the HPC platform.

### CODE REVIEW

**Comment by CoryMartin-NOAA** [comment]
This is in draft since it needs testing both manually and in cron before I would want it merged.

File: ci/stable_driver.sh [hunk 1]
```diff
@@ -117,7 +117,7 @@ if [ $ci_status -eq 0 ]; then
   fi
   if [ $total -ne 0 ]; then
     echo "Issue merging with develop. please manually fix"
-    PEOPLE="Cory.R.Martin@noaa.gov Russ.Treadon@noaa.gov Guillaume.Vernieres@noaa.gov"
+    PEOPLE="Cory.R.Martin@noaa.gov Russ.Treadon@noaa.gov Guillaume.Vernieres@noaa.gov Daniel.Holdaway@noaa.gov"
```
Comment (danholdaway) [inline ci/stable_driver.sh:120]: Disapprove 😂

**Review by danholdaway — APPROVED** [review]
Looks reasonable. Thanks for adding this, and accommodating my lazy ways of working! Helpful for quick tinkering and just having a continuous path to some modules for GDASapp that is tracking develop.

**Review by guillaumevernieres — APPROVED** [review]
Thanks @CoryMartin-NOAA

**Comment by CoryMartin-NOAA** [comment]
I'm thinking this will be superseded by stuff that Dan was working on that I hope to pick up in July (aka don't rebuild JEDI if not needed, point to existing libraries, I would think we could extend that to include this). But will not yet close this PR just in case.

**Comment by CoryMartin-NOAA** [comment]
Closing this ancient PR, @DavidNew-NOAA for awareness, this might be an area of focus to revisit soon

**Pull request closed without merging**